api/src/pcapi/routes/backoffice/offers/forms.py

```diff
@@ -96,6 +97,7 @@ class IndividualOffersAlgoliaSearchAttributes(enum.Enum):
     },
     "SYNCHRONIZED": {"field": "boolean", "operator": ["NULLABLE"]},
     "PROVIDER": {"field": "provider", "operator": ["IN", "NOT_IN"]},
+    "HEADLINE": {"field": "boolean", "operator": ["EQUALS"]},
```

Comment:
```suggestion
    "HEADLINE": {"field": "boolean", "operator": ["NULLABLE"]},
```
correspond mieux que EQUALS

Reply:
Vu ensemble sur Slack, on garde comme ça.